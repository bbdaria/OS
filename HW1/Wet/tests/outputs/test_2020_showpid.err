grep: pid: No such file or directory
grep: is: No such file or directory
grep: 34146: No such file or directory
