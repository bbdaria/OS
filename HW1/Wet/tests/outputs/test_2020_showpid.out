smash> smash pid is 34146
smash> smash pid is 34146
smash> smash> smash> smash> smash pid is 34146
smash> smash> 